src/utils/global/tooltip/tooltip.module.scss

```
@@ -0,0 +1,39 @@
+@import '../../../variables.scss';
+
+.sharedStyles {
+  background-color: $tooltip-background-color;
+  border-radius: 0.5rem;
+  color: white;
+  font-size: 1rem;
+  left: calc(100% + .75rem);
+  padding: 0.5rem;
+  position: absolute;
+  transition: $transition;
+
```

```suggestion
  transition-property: opacity;
```
Can we specify which properties we're expecting to transition? I'm assuming just opacity?